backend/mora/service/employee.py

```diff
@@ -378,6 +380,31 @@ def create_employee():
         cpr=cpr,
     )
 
+    details = util.checked_get(req, 'details', [])
+
+    decorated = _decorate_create_payloads(details, userid, valid_from,
+                                          valid_to)
+    # Validate the creation payloads
+    validated_details = writing.process_requests(
+        writing.CREATE_VALIDATION_HANDLERS,
+        decorated)
+
     userid = c.bruger.create(user, uuid=userid)
 
-    return flask.jsonify(userid)
+    creation_uuids = writing.process_requests(writing.CREATION_HANDLERS,
+                                              validated_details)
+
+    return flask.jsonify([userid] + creation_uuids)
+
+
+def _decorate_create_payloads(details, employee_uuid, valid_from, valid_to):
+    decorated = copy.deepcopy(details)
+    for detail in decorated:
+        detail['person'] = {
+            mapping.UUID: employee_uuid,
+            mapping.VALID_FROM: valid_from,
+            mapping.VALID_TO: valid_to,
+            'future': True
```

Comment:
I know I suggested `future` (on a whim) but it doesn't feel right. `allow_nonexistent`? `created`?